boards/xg23b-pk6068a/include/board.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2022 SSV Software Systems GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_xg23b-pk6068a
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the xG23B-PK6068A
+ *
+ * @author      Juergen Fitschen <me@jue.yt>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Initialize pm_layered with no pre-defined blockers
+ *
+ * All peripheral drivers keep track of required pm modes.
+ */
+#define PM_BLOCKER_INITIAL { 0, 0, 0 }
+
+/**
+ * @name    ztimer configuration
+ * @{
+ */
+#define CONFIG_ZTIMER_LPTIMER_DEV            TIMER_DEV(1)
```

Comment:
For each define that is not documented, an exclude pattern is required in `dist/tools/dockcheck/exclude_patterns`.

Personally, I hate docheck's insistence on documenting each self-explanatory define. The `explude_pattern` file is growing and growing with patterns that make no sense. We will need a solution for that problem in future.

Reply:
https://github.com/RIOT-OS/RIOT/pull/18780/commits/1581e2915d5ead8043eb5ef566e29bcef84d29ea